receptai.data/RecipePlatformDbContext.cs

```diff
@@ -39,5 +38,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             },
         ];
         modelBuilder.Entity<IdentityRole<int>>().HasData(roles);
+
+        modelBuilder.Entity<User>()
+            .HasMany(s => s.Subfooddits)
+            .WithMany(u => u.Users)
+            .UsingEntity(su => su.ToTable("SubfoodditUser"));
```

Comment:
isn't this many-to-many already created by EF? though ig being explicit never hurt anyone

Reply:
Removed redundant statement)